src/tests/test_solve_postfix_equation.py

```diff
@@ -59,13 +59,13 @@
     ("-12398.236 cos", 0.05937245447),
     ("9861236 cos", -0.95922754479),
 
-    ("0 _", 0),
-    ("1 _", 1),
-    ("43 _", 6.5574385243),
-    ("100 _", 10),
-    ("875564896227 _", 935716.247709),
-    ("1235.563 _", 35.1505760977),
-    ("59872369.547896589 _", 7737.7237963),
+    ("0 _ 1", 0),
+    ("1 _ 1", 1),
+    ("43 _ 5", 2.121747460841898),
+    ("100 _ 2", 10),
+    ("875564896227 _ 1", 875564896227),
+    ("1235.563 _ 1.2", 377.1847094765633),
+    ("59872369.547896589 _ 11.369", 4.83140055442166),
```

Comment:
sqrt by měl být ve tvaru v postfixu "x(základ) y(exponent) _", ty to máš infixově "x(základ) _ y(exponent)"